Source/WebGPU/WebGPU/Device.mm

```diff
@@ -189,6 +198,24 @@ static void captureFrame(id<MTLDevice> captureObject)
     m_isLost = true;
 }
 
+static void setOwnerWithIdentity(id<MTLResourceSPI> resource, auto webProcessID)
+{
+    ASSERT([resource respondsToSelector:@selector(setOwnerWithIdentity:)]);
+    if ([resource respondsToSelector:@selector(setOwnerWithIdentity:)]) {
+        auto result = [resource setOwnerWithIdentity:webProcessID];
+        ASSERT_UNUSED(result, result == kIOReturnSuccess);
+    }
+}
+
+void Device::setOwnerWithIdentity(id<MTLResource> resource) const
+{
+    auto webProcessID = m_adapter->webProcessID();
+    if (!webProcessID)
```

Comment:
I think this deserves a comment saying "WebKit1" or "In-process WebGPU" or something